go/vt/sqlparser/parse_test.go

```diff
@@ -4353,6 +4359,135 @@ func TestParseDjangoQueries(t *testing.T) {
+}
+
+var nonReservedKeywords = []string{"action", "active", "admin", "against", "authentication", "before", "begin", "bigint",
+	"serial", "bit", "blob", "bool", "boolean", "buckets", "cascade", "catalog_name", "change", "char", "character", "charset",
+	"check", "cipher", "class_origin", "client", "clone", "collation", "columns", "column_name", "commit", "committed", "component",
+	"constraint", "constraint_catalog", "constraint_name", "constraint_schema", "contains", "cursor_name", "data", "date", "datetime",
+	"day", "decimal", "declare", "definer", "definition", "description", "double", "duplicate", "each", "enforced", "engines", "enum",
+	"except", "exclude", "expansion", "expire", "fields", "fixed", "float_type", "flush", "foreign", "fulltext", "geomcollection",
+	"geometry", "geometrycollection", "get_master_public_key", "global", "grants", "histogram", "history", "inactive", "indexes",
+	"initial", "int", "integer", "invisible", "invoker", "isolation", "issuer", "json", "keys", "key_block_size", "language", "last_insert_id",
+	"less", "level", "lines", "linestring", "load", "local", "locked", "longblob", "longtext", "low_priority", "master_compression_algorithms",
+	"master_public_key_path", "master_tls_ciphersuites", "master_zstd_compression_level", "max_connections_per_hour", "max_queries_per_hour",
+	"max_updates_per_hour", "max_user_connections", "mediumblob", "mediumint", "mediumtext", "message_text", "mode", "modify", "multilinestring",
+	"multipoint", "multipolygon", "mysql_errno", "names", "national", "nchar", "nested", "network_namespace", "never", "no", "nowait", "nulls",
+	"numeric", "offset", "oj", "old", "only", "optimize", "option", "optional", "optionally", "ordinality", "organization", "others", "partition",
+	"path", "persist", "persist_only", "plugins", "point", "polygon", "precedes", "preceding", "precision", "primary", "privilege_checks_user",
+	"privileges", "proxy", "query", "random", "range", "read", "real", "reference", "release", "reorganize", "repair", "repeatable", "replication",
+	"require_row_format", "resignal", "resource", "respect", "restart", "restrict", "retain", "reuse", "role", "rollback", "routine", "rows",
+	"savepoint", "schemas", "schema_name", "secondary", "secondary_engine", "secondary_load", "secondary_unload", "security", "sequence",
+	"serializable", "session", "share", "signal", "signed", "skip", "slave", "smallint", "spatial", "sqlstate", "srid", "ssl", "start",
+	"starting", "stream", "subclass_origin", "subject", "tables", "tablespace", "table_name", "temporary", "text", "than", "thread_priority",
+	"ties", "time", "timestamp", "tinyblob", "tinyint", "tinytext", "transaction", "triggers", "truncate", "unbounded", "uncommitted",
+	"unsigned", "unused", "user", "varbinary", "varchar", "variables", "varying", "vcpu", "view", "visible", "warnings", "work", "write",
+	"x509", "year", "zerofil",
+}
+
+func TestKeywordsSelect(t *testing.T) {
+	t.Skip()
```

Comment:
What I want is for you to divide the lists above into 4 lists:

1) Things that should parse and do
2) Things that don't parse and should
3) Things that parse and shouldn't
4) Things that don't parse and shouldn't

Then have tests for 1) and 4) to make sure they stay that way, and skips for 2) and 3) to document the defect